README.md

```diff
@@ -108,6 +108,17 @@ includes GlobalFoundries shuttles, design contests and IC design
 workshops. The OpenROAD flow has been successfully used to date
 in over 600 silicon-ready tapeouts for technologies up to 12nm.
 
+## Getting Involved
+
+There is a users mailing list, which you can join [here](https://groups.google.com/g/openroad-users).
+
+There is also a [Matrix](https://matrix.org/) channel for discussion at 
+[#openroad:matrix.org](https://matrix.to/#/%23openroad:matrix.org).
```

Comment:
I'm also unfamiliar with this service.  It appears to require an account to even see the content so I don't know how relevant it is.  I don't believe anyone from the OR team interacts here.  Do you see a lot of useful content here?

Reply:
Being a social media ludite, haven't heard of it...